File: synthetix/runs/20260307_031713_a6e2e5e3/qa/generated_code/businesslogicservice/businesslogicservice/controllers/healthcontroller.cs
using Microsoft.AspNetCore.Mvc;

namespace BusinessLogicService.Controllers
{
    [ApiController]
    [Route("[controller]")]
    public class HealthController : ControllerBase
    {
        [HttpGet("/health")]
        public IActionResult GetHealth()
        {
            return Ok(new { status = "healthy" });
        }

        [HttpGet("/ready")]
        public IActionResult GetReady()
        {
            return Ok(new { status = "ready" });
        }
    }
}

File: synthetix/runs/20260518_140237_5e85e70a/generated_code/commentservice/src/main/kotlin/com/example/commentservice/controller/healthcontroller.kt
package com.example.commentservice.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthController {

    @GetMapping("/health")
    fun health(): Map<String, String> = mapOf("status" to "healthy")

    @GetMapping("/ready")
    fun ready(): Map<String, String> = mapOf("status" to "ready")
}